src/task_scl_eff_pot_hab.py

```diff
@@ -162,6 +116,29 @@ def density_to_patch_size(ft):
                 None, {"ECO_ID": ft.get("ECO_ID"), "min_core_size": min_core_size}
             )
 
+        def dilate(image, distance):
```

Comment:
This is awesome though a little black-box.

Reply:
a little bit black boxy, but it's nice and fast.  the distance